art/estimators/estimator.py

```diff
@@ -338,6 +338,16 @@ def compute_loss(self, x: np.ndarray, y: Any, **kwargs) -> np.ndarray:
         """
         raise NotImplementedError
 
+    def compute_loss_from_predictions(self, pred: np.ndarray, y: np.ndarray, **kwargs) -> np.ndarray:
```

Comment:
Why do we need this new method in addition to the existing method compute_losses?

Reply:
To serve the use case where we may not have access to training data (features) but only to the model predictions.